code/game/gamemodes/objective.dm

```diff
@@ -72,9 +94,9 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
   * Called when the objective's target goes to cryo.
   */
 /datum/objective/proc/on_target_cryo()
-	if(owner?.current)
-		to_chat(owner.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
-		SEND_SOUND(owner.current, sound('sound/ambience/alarm4.ogg'))
+	for(var/datum/mind/M in get_owners())
+		to_chat(M.current, "<BR><span class='userdanger'>You get the feeling your target is no longer within reach. Time for Plan [pick("A","B","C","D","X","Y","Z")]. Objectives updated!</span>")
+		SEND_SOUND(M.current, sound('sound/ambience/alarm4.ogg'))
 	target = null
 	INVOKE_ASYNC(src, .proc/post_target_cryo)
```

Comment:
to save doing `get_owners` twice couldnt you cashe it and pass it to `post_target_cryo` not sure if its called anywhere else